WordPress/Classes/ViewRelated/NUX/LoginViewController.swift

```diff
@@ -117,7 +117,11 @@ extension LoginViewController: SigninWPComSyncHandler, LoginFacadeDelegate {
             return
         }
 
-        displayError(error as NSError, sourceTag: sourceTag)
+        let presentingController = navigationController ?? self
```

Comment:
My only real change request: I think this should probably go into `displayError(_ error: NSError, sourceTag: SupportSourceTag)`. This is called from a few places still, like `LoginLinkRequestViewController` ln 108.

I _think_ that'll be a pretty straight forward change tho.